Native script parsing error in Preview

There is an error related to native script parsing in Preview
Block 3300404

```
java.lang.RuntimeException: com.bloxbean.cardano.client.exception.CborRuntimeException: Error parsing native script
	at com.bloxbean.cardano.yaci.store.core.service.BlockFetchService.onParsingError(BlockFetchService.java:349) ~[main/:na]
	at com.bloxbean.cardano.yaci.helper.listener.BlockFetchAgentListenerAdapter.onParsingError(BlockFetchAgentListenerAdapter.java:137) ~[yaci-helper-0.4.0-beta2.jar:na]
	at com.bloxbean.cardano.yaci.core.protocol.blockfetch.BlockfetchAgent.lambda$onReceiveBlocks$5(BlockfetchAgent.java:150) ~[yaci-core-0.4.0-beta2.jar:na]
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1708) ~[na:na]
	at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762) ~[na:na]
	at com.bloxbean.cardano.yaci.core.protocol.blockfetch.BlockfetchAgent.onReceiveBlocks(BlockfetchAgent.java:150) ~[yaci-core-0.4.0-beta2.jar:na]
	at com.bloxbean.cardano.yaci.core.protocol.blockfetch.BlockfetchAgent.processResponse(BlockfetchAgent.java:83) ~[yaci-core-0.4.0-beta2.jar:na]
	at com.bloxbean.cardano.yaci.core.protocol.Agent.receiveResponse(Agent.java:44) ~[yaci-core-0.4.0-beta2.jar:na]
	at com.bloxbean.cardano.yaci.core.network.handlers.MiniProtoClientInboundHandler.channelRead(MiniProtoClientInboundHandler.java:39) ~[yaci-core-0.4.0-beta2.jar:na]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:346) ~[netty-codec-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:333) ~[netty-codec-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.handler.codec.ReplayingDecoder.callDecode(ReplayingDecoder.java:349) ~[netty-codec-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290) ~[netty-codec-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1357) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:868) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562) ~[netty-transport-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.113.Final.jar:4.1.113.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.113.Final.jar:4.1.113.Final]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[na:na]
Caused by: com.bloxbean.cardano.client.exception.CborRuntimeException: Error parsing native script
	at com.bloxbean.cardano.yaci.core.model.serializers.WitnessesSerializer.deserializeNativeScript(WitnessesSerializer.java:226) ~[yaci-core-0.4.0-beta2.jar:na]
	at com.bloxbean.cardano.yaci.core.model.serializers.WitnessesSerializer.deserializeDI(WitnessesSerializer.java:55) ~[yaci-core-0.4.0-beta2.jar:na]
	at com.bloxbean.cardano.yaci.core.model.serializers.BlockSerializer.deserializeBlock(BlockSerializer.java:77) ~[yaci-core-0.4.0-beta2.jar:na]
	at com.bloxbean.cardano.yaci.core.model.serializers.BlockSerializer.deserialize(BlockSerializer.java:32) ~[yaci-core-0.4.0-beta2.jar:na]
	at com.bloxbean.cardano.yaci.core.protocol.blockfetch.BlockfetchAgent.onReceiveBlocks(BlockfetchAgent.java:128) ~[yaci-core-0.4.0-beta2.jar:na]
	... 26 common frames omitted
```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -26,7 +26,7 @@ jobs:
           echo "${{secrets.SIGNING_KEY}}" > ~/.gradle/secring.gpg.b64
           base64 -d ~/.gradle/secring.gpg.b64 > ~/.gradle/secring.gpg
       - name: Publish package
-        run: ./gradlew publish -Psigning.keyId=${{ secrets.SIGNING_KEY_ID }} -Psigning.password=${{ secrets.SIGNING_PASSWORD }} -Psigning.secretKeyRingFile=$(echo ~/.gradle/secring.gpg) --warn --stacktrace
+        run: ./gradlew publishToSonatype closeSonatypeStagingRepository -Psigning.keyId=${{ secrets.SIGNING_KEY_ID }} -Psigning.password=${{ secrets.SIGNING_PASSWORD }} -Psigning.secretKeyRingFile=$(echo ~/.gradle/secring.gpg) --warn --stacktrace
         env:
           MAVEN_USERNAME: ${{ secrets.OSSRH_USERNAME }}
           MAVEN_PASSWORD: ${{ secrets.OSSRH_TOKEN }}

--- a/build.gradle
+++ b/build.gradle
@@ -6,6 +6,7 @@ plugins {
     id 'io.spring.dependency-management' version "${springDependencyManagementVersion}"
 
     id 'org.ajoberstar.grgit' version '5.2.0'
+    id 'io.github.gradle-nexus.publish-plugin' version '2.0.0'
 }
 
 subprojects {
@@ -17,7 +18,7 @@ subprojects {
         mavenLocal()
         mavenCentral()
         maven {
-            url "https://oss.sonatype.org/content/repositories/snapshots"
+            url "https://central.sonatype.com/repository/maven-snapshots/"
         }
     }
 
@@ -88,4 +89,32 @@ subprojects {
 
 def getCheckedOutGitCommitHash() {
     grgit.head().abbreviatedId
+}
+
+ext.ossrhUsername = System.getenv('MAVEN_USERNAME')
+ext.ossrhPassword = System.getenv('MAVEN_PASSWORD')
+
+publishing {
+    repositories {
+        maven {
+            def releasesRepoUrl = "https://ossrh-staging-api.central.sonatype.com/service/local/staging/deploy/maven2/"
+            def snapshotsRepoUrl = "https://central.sonatype.com/repository/maven-snapshots/"
+            url = version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
+            credentials {
+                username ossrhUsername
+                password ossrhPassword
+            }
+        }
+    }
+}
+
+nexusPublishing {
+    repositories {
+        sonatype {
+            nexusUrl.set(uri("https://ossrh-staging-api.central.sonatype.com/service/local/"))
+            snapshotRepositoryUrl.set(uri("https://central.sonatype.com/repository/maven-snapshots/"))
+            username = ossrhUsername
+            password = ossrhPassword
+        }
+    }
 }

--- a/gradle.properties
+++ b/gradle.properties
@@ -1,5 +1,5 @@
 group = com.bloxbean.cardano
-version = 0.1.2
+version = 0.1.3-beta
 
 springBootVersion=3.3.4
 springCloudVersion=2023.0.3

--- a/gradle.properties
+++ b/gradle.properties
@@ -1,5 +1,5 @@
 group = com.bloxbean.cardano
-version = 0.1.3-beta
+version = 0.1.3
 
 springBootVersion=3.3.4
 springCloudVersion=2023.0.3

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -1,8 +1,8 @@
 [libraries]
-yaci = "com.bloxbean.cardano:yaci:0.3.5"
-cardano-client-lib = "com.bloxbean.cardano:cardano-client-lib:0.6.2"
-cardano-client-backend = "com.bloxbean.cardano:cardano-client-backend:0.6.2"
-cardano-client-backend-ogmios = "com.bloxbean.cardano:cardano-client-backend-ogmios:0.6.2"
+yaci = "com.bloxbean.cardano:yaci:0.3.6"
+cardano-client-lib = "com.bloxbean.cardano:cardano-client-lib:0.6.6"
+cardano-client-backend = "com.bloxbean.cardano:cardano-client-backend:0.6.6"
+cardano-client-backend-ogmios = "com.bloxbean.cardano:cardano-client-backend-ogmios:0.6.6"
 
 rocks-types="com.bloxbean:rocks-types:0.0.1-preview6"
 

--- a/publish-common.gradle
+++ b/publish-common.gradle
@@ -39,21 +39,6 @@ if (!name.equalsIgnoreCase("stores") && !name.equalsIgnoreCase("components")) {
                 }
             }
         }
-
-        repositories {
-            String ossrhUsername = System.getenv('MAVEN_USERNAME')
-            String ossrhPassword = System.getenv('MAVEN_PASSWORD')
-
-            maven {
-                def releasesRepoUrl = "https://oss.sonatype.org/service/local/staging/deploy/maven2"
-                def snapshotsRepoUrl = "https://oss.sonatype.org/content/repositories/snapshots/"
-                url = version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
-                credentials {
-                    username ossrhUsername
-                    password ossrhPassword
-                }
-            }
-        }
     }
 }
 

--- a/starters/build.gradle
+++ b/starters/build.gradle
@@ -43,21 +43,6 @@ subprojects {
                     }
                 }
             }
-
-            repositories {
-                String ossrhUsername = System.getenv('MAVEN_USERNAME')
-                String ossrhPassword = System.getenv('MAVEN_PASSWORD')
-
-                maven {
-                    def releasesRepoUrl = "https://oss.sonatype.org/service/local/staging/deploy/maven2"
-                    def snapshotsRepoUrl = "https://oss.sonatype.org/content/repositories/snapshots/"
-                    url = version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
-                    credentials {
-                        username ossrhUsername
-                        password ossrhPassword
-                    }
-                }
-            }
         }
     }
 

